csharp/src/Apache.Arrow/Arrays/ListArray.cs

```diff
@@ -135,6 +152,11 @@ public int GetValueOffset(int index)
 
         public int GetValueLength(int index)
         {
+            if (IsNull(index))
```

Comment:
This should be after the argument checking.

Reply:
Fixed